src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/TagHelperCompletionProvider.cs

```diff
@@ -17,11 +18,12 @@ namespace Microsoft.AspNetCore.Razor.LanguageServer.Completion
     internal class TagHelperCompletionProvider : RazorCompletionItemProvider
     {
         // Internal for testing
-        internal static readonly IReadOnlyCollection<string> MinimizedAttributeCommitCharacters = new List<string> { "=", " " };
-        internal static readonly IReadOnlyCollection<string> AttributeCommitCharacters = new List<string> { "=" };
+        internal static readonly IReadOnlyList<RazorCommitCharacter> MinimizedAttributeCommitCharacters = new[] { "=", " " }.Select(c => new RazorCommitCharacter(c)).ToArray();
+        internal static readonly IReadOnlyList<RazorCommitCharacter> AttributeCommitCharacters = new[] { "=" }.Select(c => new RazorCommitCharacter(c)).ToArray();
+        internal static readonly IReadOnlyList<RazorCommitCharacter> AttributeSnippetCommitCharacters = new[] { "=" }.Select(c => new RazorCommitCharacter(c, Insert: false)).ToArray();
```

Comment:
If anyone else is looking, the `Insert: false` here is the entire justification for this PR it seems 😛

Reply:
Yup 😄!